### PR TITLE
add RT_Dino

### LINKED ISSUE #33: RT-Dino
Resources
[Highly-Recommended Guide to Follow](https://karpathy.github.io/2019/04/25/recipe/?fbclid=IwAR0HcxdKV2lPGYGKTZI7x1wkNAnxfp7hvNHAHPJl2CtxAK26w44d76n37ig)

https://github.com/lucidrains/vit-pytorch?tab=readme-ov-file#3d-vit
[DinoV2 Source Code](https://github.com/facebookresearch/dinov2)
https://github.com/lucidrains/classifier-free-guidance-pytorch

[MinMax Scaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#minmaxscaler)
[kbins](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.KBinsDiscretizer.html)

DinoV2 backbone, multihead transformer head, classifier free film layers, xarm data
- [ ] Use ViT-g/14 with registers 
- [ ] Asymmetric loss: https://timm.fast.ai/asymmetric_loss
- [ ] CutUpMix: https://timm.fast.ai/random_resized_crop
- [ ] https://timm.fast.ai/random_resized_crop
- [ ] [PCA] (https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html#sklearn.decomposition.PCA)

--

<br class="Apple-interchange-newline">

### PR DESCRIPTION
**Pull request by emekaokoli19**
This pull request aims to resolve #33

### CODE REVIEW

**Comment by emekaokoli19** [comment]
@sebbyjp I want to work on #33 
I'll let you know when I have something to review, for now this PR will be draft

**Comment by emekaokoli19** [comment]
> Looking forward to it!

Hi @sebbyjp please take a look when you have the chance.
